src/main/java/frc/team2412/robot/commands/intake/IntakeInCommand.java

```diff
@@ -5,7 +5,7 @@
 
 public class IntakeInCommand extends CommandBase {
 
-    public final IntakeSubsystem subsystem;
+    protected final IntakeSubsystem subsystem;
 
     public IntakeInCommand(IntakeSubsystem subsystem) {
         this.subsystem = subsystem;
```

Comment:
While you're here, please move the ` subsystem.intakeIn();` from execute() to initialize().  There's no need to waste time every 20ms re-setting the intake motors to a speed they're already running at.